src/services/MakeStreamZip.php

```diff
@@ -148,16 +147,18 @@ private function nameFolder(): void
     /**
      * Folder and zip file name begins with date for experiments
      *
-     * @return void
+     * @return string
      */
     private function getBaseFileName(): string
     {
         if ($this->Entity instanceof Experiments) {
             return $this->Entity->entityData['date'] . ' - ' . Filter::forFilesystem($this->Entity->entityData['title']);
         } elseif ($this->Entity instanceof Database) {
             return $this->Entity->entityData['category'] . ' - ' . Filter::forFilesystem($this->Entity->entityData['title']);
+        } else {
```

Comment:
no need for else here. just throw directly.

Reply:
Ok, I will change that.